assets/overlays/openstack-cinder/generated/hypershift/controller.yaml

```diff
@@ -164,17 +169,11 @@ spec:
         - --leader-election-namespace=${NODE_NAMESPACE}
         - --v=${LOG_LEVEL}
         - --timeout=3m
-        - --feature-gates=Topology=$(ENABLE_TOPOLOGY)
```

Comment:
Removing this is fine, because we're now achieving the same outcome by nobbling the behaviour of `CreateVolume` when `enable_topology` is false.